iotdb-client/client-py/TableModelSessionPoolExample.py

```diff
@@ -0,0 +1,160 @@
+        "CREATE TABLE table1 (id1 string id, attr1 string attribute, "
+        + "m1 double "
+        + "measurement)"
+    )
+
+    print("now the tables are:")
+    # show result
+    res = session.execute_query_statement("SHOW TABLES")
+    while res.has_next():
+        print(res.next())
+
+    session_pool.put_back(session)
+
+
+def insert_data(num: int):
+    print("insert data for table" + str(num))
+    # Get a session from the pool
+    session = session_pool.get_session()
+    column_names = [
+        "id1",
+        "attr1",
+        "m1",
+    ]
+    data_types = [
+        TSDataType.STRING,
+        TSDataType.STRING,
+        TSDataType.DOUBLE,
+    ]
+    column_types = [ColumnType.ID, ColumnType.ATTRIBUTE, ColumnType.MEASUREMENT]
+    timestamps = []
+    values = []
+    for row in range(15):
+        timestamps.append(row)
+        values.append(["id:" + str(row), "attr:" + str(row), row * 1.0])
+    tablet = Tablet(
+        "table" + str(num), column_names, data_types, values, timestamps, column_types
+    )
+    session.insert_relational_tablet(tablet)
+    session.execute_non_query_statement("FLush")
+
+    np_timestamps = np.arange(15, 30, dtype=np.dtype(">i8"))
+    np_values = [
+        np.array(["id:{}".format(i) for i in range(15, 30)]),
+        np.array(["attr:{}".format(i) for i in range(15, 30)]),
+        np.linspace(15.0, 29.0, num=15, dtype=TSDataType.DOUBLE.np_dtype()),
+    ]
+
+    np_tablet = NumpyTablet(
+        "table" + str(num),
+        column_names,
+        data_types,
+        np_values,
+        np_timestamps,
+        column_types=column_types,
+    )
+    session.insert_relational_tablet(np_tablet)
+    session_pool.put_back(session)
+
+
+def query_data():
+    # Get a session from the pool
+    session = session_pool.get_session()
+
+    print("get data from table0")
+    res = session.execute_query_statement("select * from table0")
+    while res.has_next():
+        print(res.next())
+
+    print("get data from table1")
+    res = session.execute_query_statement("select * from table0")
+    while res.has_next():
+        print(res.next())
+
+    session_pool.put_back(session)
+
+
+def delete_data():
+    session = session_pool.get_session()
+    session.execute_non_query_statement("drop database db1")
+    print("data has been deleted. now the devices are:")
+    res = session.execute_statement("show databases")
+    while res.has_next():
+        print(res.next())
+    session_pool.put_back(session)
+
+
+ip = "127.0.0.1"
+port = "6667"
+username = "root"
+password = "root"
+pool_config = PoolConfig(
+    node_urls=["127.0.0.1:6667", "127.0.0.1:6668", "127.0.0.1:6669"],
+    user_name=username,
+    password=password,
+    fetch_size=1024,
+    time_zone="UTC+8",
+    max_retry=3,
+    sql_dialect="table",
+    database="db1",
+)
+max_pool_size = 5
+wait_timeout_in_ms = 3000
+
+# Create a session pool
+session_pool = SessionPool(pool_config, max_pool_size, wait_timeout_in_ms)
```

Comment:
Just curious: what is the standard of putting configs in `pool_config` or not? ` max_pool_size` and `wait_timeout_in_ms` seem perfectly fit in `pool_config` but they are listed separately.

Reply:
Don’t know +1. @ycycse do you have any idea?

Reply:
> Don’t know +1. @ycycse do you have any idea?

Configs in `pool_config` is used to construct and connect session which contains relating infos about session. I think change it to `session_config` would be more clear.